schema/crdb/10.0.0/up1.sql

```diff
@@ -0,0 +1 @@
+CREATE SEQUENCE IF NOT EXISTS omicron.public.ipv4_nat_version START 1 INCREMENT 1;
```

Comment:
Before you merge this -- looks like `10.0.0` is already used on main?

https://github.com/oxidecomputer/omicron/tree/main/schema/crdb/10.0.0

Reply:
Awaiting merge of https://github.com/oxidecomputer/dendrite/pull/627

Reply:
Dendrite PR merged, bumping Omicron dependency version and Schema version